master/docs/manual/cfg-dbconfig.rst

```diff
@@ -0,0 +1,42 @@
+.. bb:cfg:: dbconfig
+
+
+DbConfig
+--------
+
+DbConfig is an utility for master.cfg to get easy key/value storage in the buildbot database
+
+DbConfig can get and store any jsonable object to the db for use by other masters or separate ui plugins to edit them.
+
+The design is voluntary simplistic, the focus is on the easy use.
+
+Example ::
+
+    from buildbot.plugins import util, buildslave
+    c = BuildmasterConfig = {}
+    c['db_url'] = 'mysql://user@pass:mysqlserver/buildbot'
+    dbConfig = util.DbConfig(BuildmasterConfig, basedir)
+    slaves = dbConfig.get("slaves")
+    c['slaves'] = [
+        buildslave.BuildSlave(slave['name'], slave['passwd'],
+                              properties=slave.get('properties')),
+        for slave in slaves
+    ]
+
```

Comment:
again answering to self: no, the database is opened and closed for every get/set.